api/src/server.rs

```diff
@@ -254,6 +269,24 @@ where
             .collect();
         Ok(SubscribeSortKeySortValueGetResponse::Success(events))
     }
+
+    #[instrument(skip(self, _context), ret(level = Level::DEBUG), err(level = Level::ERROR))]
+    async fn interests_sort_key_sort_value_post(
+        &self,
+        sort_key: String,
+        sort_value: String,
+        controller: Option<String>,
+        stream_id: Option<String>,
+        _context: &C,
+    ) -> Result<InterestsSortKeySortValuePostResponse, ApiError> {
+        debug!(
```

Comment:
nit, this debug is redundant as the params are already part of the tracing context and will get logged at debug when the function returns successfully or logged at error in the case of an error.

Reply:
Ah nice, ok, taking it out.